src/Symfony/Bundle/FrameworkBundle/Resources/config/services.xml

```diff
@@ -87,7 +87,7 @@
 
         <service id="file_locator" class="Symfony\Component\HttpKernel\Config\FileLocator">
             <argument type="service" id="kernel" />
-            <argument>%kernel.root_dir%/Resources</argument>
```

Comment:
you removed this argument. how does `%kernel.root_dir%/Resources` still work then?